src/urllib3/connectionpool.py

```diff
@@ -364,8 +353,7 @@ def _make_request(
         # Trigger any extra validation we need to do.
         try:
             self._validate_conn(conn)
-        except (SocketTimeout, BaseSSLError) as e:
```

Comment:
I'm wondering if there's a way that `BaseSSLError` could be raised here that we're not aware of?

Reply:
@sethmlarson I will check it.

Reply:
@sethmlarson 
I checked it and couldn't find a way for `BaseSSLError` but we can keep `BaseSSLError` if you prefer.

Reply:
Let's keep the `BaseSSLError` in case there's "bad MAC" errors we could catch here.

Reply:
Done!